tests/rados/test_osd_onode_trimming.py

```diff
@@ -0,0 +1,149 @@
+"""
+Method to verify the onode trimming
+"""
+import datetime
+import time
+
+from ceph.ceph_admin import CephAdmin
+from ceph.rados.core_workflows import RadosOrchestrator
+from tests.rados.monitor_configurations import MonConfigMethods
+from utility.log import Log
+from utility.utils import method_should_succeed
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    log.info(run.__doc__)
+    config = kw["config"]
+    cephadm = CephAdmin(cluster=ceph_cluster, **config)
+    rados_object = RadosOrchestrator(node=cephadm)
+    mon_obj = MonConfigMethods(rados_obj=rados_object)
+    ceph_nodes = kw.get("ceph_nodes")
+    installer = ceph_cluster.get_nodes(role="installer")[0]
+    pool_name = "test-osdonode"
+    log.info("Running test case to verify trimming of onodes")
+
+    osd_list = []
+    onode_values = {}
+    log_line = [
+        "trim maximum skip pinned reached",
+    ]
```

Comment:
Check the default value of the param, and if not 1000, ( default ) , fail it

Reply:
Implemented the code.